client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1792,6 +1792,10 @@ Undocumented.prototype.applyDnsTemplate = function( domain, template, variables,
 	return this.wpcom.req.post( '/domains/' + domain + '/dns/template/' + template, { variables }, callback );
 };
 
+Undocumented.prototype.getDnsTemplateConflicts = function( domain, template, variables, callback ) {
+	return this.wpcom.req.post( '/domains/' + domain + '/dns/template/' + template + '/conflicts', { variables }, callback );
```

Comment:
I'm sure you were influenced how some of the other functions in this file were written, but just FYI - `this.wpcom.req.post` (and `get`, of course) already return a `Promise`, so you could use that instead of an explicit (and less robust and more verbose) callback.